### PR TITLE
feat: Add translations for earn cards in home page

### PR DESCRIPTION
**Pull request by memoyil**
@hachiojidev Sorry for the inconvenience

### CODE REVIEW

File: src/views/Home/components/EarnAPRCard.tsx [hunk 1]
```diff
@@ -50,19 +50,23 @@ const EarnAPRCard = () => {
     return maxApr?.toLocaleString('en-US', { maximumFractionDigits: 2 })
   }, [cakePrice, farmsLP, prices])
 
+  const aprText = highestApr || '#placeholder#'
+  const earnAprText = t('Earn up to %highestApr% APR in Farms', { highestApr: aprText })
+  const aprTextParts = earnAprText.split(aprText)
```
Comment (hachiojidev) [inline src/views/Home/components/EarnAPRCard.tsx:55]: Can we deconstruct this.
```
const [earnUpTo, InFarms] = earnAprText.split(aprText)
```

File: src/views/Home/components/EarnAPRCard.tsx [hunk 2]
```diff
@@ -50,19 +50,23 @@ const EarnAPRCard = () => {
     return maxApr?.toLocaleString('en-US', { maximumFractionDigits: 2 })
   }, [cakePrice, farmsLP, prices])
 
+  const aprText = highestApr || '#placeholder#'
```
Comment (hachiojidev) [inline src/views/Home/components/EarnAPRCard.tsx:53]: Lets use `-`

File: src/views/Home/components/EarnAssetCard.tsx [hunk 3]
```diff
@@ -31,17 +32,21 @@ const latestPools: Pool[] = orderBy(activeNonCakePools, ['sortOrder', 'pid'], ['
 const assets = ['CAKE', ...latestPools.map((pool) => pool.earningToken.symbol)].join(', ')
 
 const EarnAssetCard = () => {
+  const { t } = useTranslation()
+  const assetText = t('Earn %assets% in Pools', { assets })
+  const assetTextParts = assetText.split(assets)
```
Comment (hachiojidev) [inline src/views/Home/components/EarnAssetCard.tsx:37]: Same here, let's deconstruct for readability.

**Review by hachiojidev — CHANGES_REQUESTED** [review]

File: src/views/Home/components/EarnAPRCard.tsx (hunk 2 above)
Comment (memoyil) [reply src/views/Home/components/EarnAPRCard.tsx]: Done

File: src/views/Home/components/EarnAssetCard.tsx (hunk 3 above)
Comment (memoyil) [reply src/views/Home/components/EarnAssetCard.tsx]: Done

File: src/views/Home/components/EarnAPRCard.tsx (hunk 1 above)
Comment (memoyil) [reply src/views/Home/components/EarnAPRCard.tsx]: done

**Review by hachiojidev — APPROVED** [review]

**Pull request merged**